zhaquirks/lixee/zlinky.py

```diff
@@ -15,6 +16,19 @@
 )
 from zhaquirks.lixee import LIXEE
 
+ZLINKY_MANUFACTURER_CLUSTER_ID = 0xFF66
```

Comment:
Maybe this constant can be defined in the `__init__.py` and then line 17 updated to ` from zhaquirks.lixee import LIXEE, ZLINKY_MANUFACTURER_CLUSTER_ID`

Reply:
Done.